### ISSUE TITLE
Huge weight decay rate makes no difference

### ISSUE THREAD

**Issue by odysseus0** [fastai/fastai#1806]
Please refer to the notebook below for more details:

https://nbviewer.jupyter.org/gist/odysseus0/c24de188f3266b30def9c6f5771d4e08

I am not sure if such behavior is normal or not. Need your wisdom.

**Comment by sgugger** [comment]
Try 1000, it will go nan in one epoch ;)

**Comment by odysseus0** [comment]
Yeah, I know what you mean. I tried that and it did make a difference.

The issue is that usually when there is overfitting, you jack up the wd to reduce it. However, with our particular setup, I did not find a logarithmically larger wd makes much difference. Probably I am doing it wrong.
